src/main/java/com/laytonsmith/core/functions/ArrayHandling.java

```diff
@@ -130,6 +130,9 @@ public String getName() {
 
 		@Override
 		public Construct exec(Target t, Environment env, Construct... args) throws CancelCommandException, ConfigRuntimeException {
+			if(args.length == 0) {
+				throw new ConfigRuntimeException("Argument 1 of array_get must be an array", ExceptionType.CastException, t);
```

Comment:
In general, it is better to use getName() instead of hardcoding the name of the function.